lib/solaar/ui/window.py

```diff
@@ -593,7 +593,7 @@ def _update_receiver_panel(receiver, panel, buttons, full=False):
 
 	may_pair = receiver.may_unpair and not is_pairing
 	if may_pair and devices_count >= receiver.max_devices:
-		online_devices = tuple(n for n in range(1, receiver.max_devices) if n in receiver and receiver[n].online)
+		online_devices = tuple(n for n in range(1, receiver.max_devices+1) if n in receiver)
```

Comment:
After the removal of the online check, it is in my view more clear to renamed the variable to `paired_devices` instead of `online_devices`

Reply:
Agreed.  
I'm not even sure that there is any need for iterating over the devices, but I'm not completely sure that it is unncessary.  I'll change the variable name and put in a comment.